src/agenda/index.js

```diff
@@ -47,6 +47,8 @@ export default class AgendaView extends Component {
     renderItem: PropTypes.func,
     // specify how each date should be rendered. day can be undefined if the item is not first in that day.
     renderDay: PropTypes.func,
+    // determine if day should be displayed above or beside item
+    dayColumn: PropTypes.bool,
```

Comment:
I don't like this prop name, it's confusing. How about changing it to something like 'columDisplay' or 'columnDayDisplay'?